src/models/subject.ts

```diff
@@ -0,0 +1,157 @@
+import { Classification } from "./decisions/classification";
+import { IDecision, VerificationDecision } from "./decisions/decision";
+import { Verification } from "./decisions/verification";
+
+export type SubjectData = Record<string, unknown>;
+
+export interface ISubjectWrapper {
+    subject: Subject;
+    identifier: symbol;
+    clientCached: boolean;
+    serverCached: boolean;
+
+    url: string;
+    decisions: Map<symbol, IDecision[]>;
+    tag: string;
+}
+
+
+export interface Tag {
+    id?: number;
+    text: string;
+}
+
+export class Subject {
+    public constructor(data: SubjectData) {
+        this.data = data;
+        this.identifier = Symbol("verification-subject");
+    }
+
+    public identifier: symbol;
+    public data: SubjectData;
+}
```

Comment:
I ignored this last time, but it seems a lot to create a wrapper object for another object just to stick an unique identifier on an object in a dynamic language.

```
interface ISubject  = unknown & { identifier: Symbol }
```

Also document the purpose of the identifier symbol (equality i believe?)

Reply:
I originally implemented this change like

![image](https://github.com/user-attachments/assets/2a7819e6-e83c-4b5e-ab2a-a87f04ec11ca)

But I'm thinking of reverting the change because `SubjectData` is user defined.
This means that if we add our identifier to the original row, the user cannot have a column named `identifier`.

We could namespace it (or other solutions) to reduce the risk of key collisions.
But I think adding a symbol to the user's input data row goes against our "we do not modify the original data" pitch

Reply:
Discussed offline:

Using a symbol as a key on the `subjectData` solves these concerns